src/commons_codec/model.py

```diff
@@ -11,28 +12,25 @@
 else:
     from backports.strenum import StrEnum  # pragma: no cover
 
+try:
+    from sqlalchemy_cratedb.support import quote_relation_name
```

Comment:
Under what circumstances will this import cause an `ImportError`, if that's the case, `TableAdress.fqn()` will cause a runtime error `TypeError: 'NoneType' object is not callable` wouldn't it?

Reply:
Yeah, unfortunate code stacking here, see https://github.com/crate/commons-codec/pull/38#discussion_r1738294810. Let's get rid of code smells completely on behalf of another iteration.

Reply:
I've amended the patch according to you suggestions, it is much more straight-forward now. Thank you!